Tile38 panics after running AOFSHRINK

Sometimes, after executing the AOFSHIRNK command the server panics and crashes.

I'm using the docker image `tile38/tile38:1.19.0` on GKE.

Logs:
```
2019/11/17 06:14:46 [INFO] aof shrink ended 162.405µs
panic: interface conversion: interface {} is int64, not time.Time

goroutine 2709 [running]:
github.com/tidwall/tile38/internal/server.(*Server).aofshrink.func2.2.1(0xc000562520, 0x11, 0xf16ae0, 0xc000470a80, 0xc00048e600, 0x2, 0x2, 0x0)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/internal/server/aofshrink.go:126 +0x14d0
github.com/tidwall/tile38/internal/collection.(*Collection).ScanGreaterOrEqual.func1(0xc000562520, 0x11, 0xc49860, 0xc000470aa0, 0xc0004ed200)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/internal/collection/collection.go:485 +0xea
github.com/tidwall/tile38/vendor/github.com/tidwall/tinybtree.(*node).ascend(0xc000489400, 0x0, 0x0, 0xc000231c08, 0x0, 0xe38c56da2931dc09)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/vendor/github.com/tidwall/tinybtree/btree.go:320 +0xc3
github.com/tidwall/tile38/vendor/github.com/tidwall/tinybtree.(*BTree).Ascend(...)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/vendor/github.com/tidwall/tinybtree/btree.go:302
github.com/tidwall/tile38/internal/collection.(*Collection).ScanGreaterOrEqual(0xc0004934a0, 0x0, 0x0, 0xcec800, 0x0, 0x0, 0x0, 0xc000197d18, 0x0)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/internal/collection/collection.go:491 +0x17f
github.com/tidwall/tile38/internal/server.(*Server).aofshrink.func2.2(0xc000197dd7, 0xc0001e8c80, 0xc000197e98, 0xc000197e50, 0xc000197e80, 0xc000197ec8)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/internal/server/aofshrink.go:103 +0x2bf
github.com/tidwall/tile38/internal/server.(*Server).aofshrink.func2(0xc0001e8c80, 0x0, 0x0)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/internal/server/aofshrink.go:159 +0x1b8
github.com/tidwall/tile38/internal/server.(*Server).aofshrink(0xc0001e8c80)
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/internal/server/aofshrink.go:301 +0x151
created by github.com/tidwall/tile38/internal/server.(*Server).command
	/tmp/tile38.nugV/go/src/github.com/tidwall/tile38/internal/server/server.go:1032 +0x4719
```

--- a/internal/server/aofshrink.go
+++ b/internal/server/aofshrink.go
@@ -98,8 +98,8 @@ func (server *Server) aofshrink() {
 					if value, ok := server.expires.Get(keys[0]); ok {
 						exm = value.(*rhh.Map)
 					}
-					var now = time.Now() // used for expiration
-					var count = 0        // the object count
+					var now = time.Now().UnixNano() // used for expiration
+					var count = 0                   // the object count
 					col.ScanGreaterOrEqual(nextid, false, nil, nil,
 						func(id string, obj geojson.Object, fields []float64) bool {
 							if count == maxids {
@@ -123,7 +123,7 @@ func (server *Server) aofshrink() {
 							}
 							if exm != nil {
 								if at, ok := exm.Get(id); ok {
-									expires := at.(time.Time).Sub(now)
+									expires := at.(int64) - now
 									if expires > 0 {
 										values = append(values, "ex")
 										values = append(values, strconv.FormatFloat(math.Floor(float64(expires)/float64(time.Second)*10)/10, 'f', -1, 64))

